assembl/alembic/versions/c3f8bc9c75d5_column_synthesis_post.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+"""Migrate IdeaMessageColumn.header field to ColumnSynthesisPost
+
+Revision ID: c3f8bc9c75d5
+Revises: c5754a7cb6be
+Create Date: 2017-12-12 09:50:52.192838
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'c3f8bc9c75d5'
+down_revision = 'c5754a7cb6be'
+
+from alembic import context, op
+import sqlalchemy as sa
+import transaction
+
+
+from assembl.lib import config
+from assembl.lib.sqla import mark_changed
+
+
+def upgrade(pyramid_env):
+    # Do stuff with the app's models here.
+    from assembl import models as m
+    db = m.get_session_maker()()
+    with transaction.manager:
+        # take the first sysadmin as creator
+        creator_id = m.User.default_db.query(m.User).join(m.User.roles).filter(m.Role.id == 7)[0:1][0].id
```

Comment:
`[0:1][0]` !?

Reply:
[0:1] will do a offset 0 limit 1 in the sql query.

Reply:
Using `7` as the role.id also feels wrong. I suspect it won't be an issue across servers due to how the servers are bootstrapped, but the query should still be stable. Find the ID of of the `r:sysadmin` and apply here. And after your comment, I still do not understand the `[0:1][0]`.

Reply:
I think `[0:1][0]` is exactly the same as `.first()` actually, does `first()` add a limit 1 in the query ?

Reply:
I did the change about the role id (on my instance, sysadmin id was 4)